scipy/stats/_continuous_distns.py

```diff
@@ -647,10 +648,10 @@ class burr_gen(rv_continuous):
 
     """
     def _pdf(self, x, c, d):
-        return c*d*(x**(-c-1.0))*((1+x**(-c*1.0))**(-d-1.0))
+        return c*d*(x**(-c-1.0))*((1+x**(-c))**(-d-1.0))
 
     def _cdf(self, x, c, d):
-        return (1+x**(-c*1.0))**(-d**1.0)
+        return (1+x**(-c))**(-d)
```

Comment:
Is there a guideline for this kind of thing?  Some reviewers seem to want contributors to smoosh all of their commits together per PR.